httpx/models.py

```diff
@@ -309,7 +309,7 @@ def __repr__(self) -> str:
         return f"{class_name}({query_string!r})"
 
 
-class Headers(typing.MutableMapping[str, str]):
+class Headers(typing.MutableMapping[str, str], MutableMapping):
```

Comment:
Why is this needed?

Reply:
I had added it thinking it would inherit the update function, but it turns out that's not how it worked. Removed it now.